src/main/java/frc/robot/subsystems/shooter/ShooterFlywheelIOSparkMax.java

```diff
@@ -0,0 +1,61 @@
+package frc.robot.subsystems.shooter;
+
+import com.revrobotics.CANSparkMax;
+import com.revrobotics.RelativeEncoder;
+import com.revrobotics.CANSparkLowLevel.MotorType;
+
+import frc.robot.Constants;
+
+public class ShooterFlywheelIOSparkMax implements ShooterFlywheelIO
+{
+    private CANSparkMax     _upperFlywheelSparkMax;
+    private CANSparkMax     _lowerFlywheelSparkMax;
+    private RelativeEncoder _upperFlywheelEncoder;
+    private RelativeEncoder _lowerFlywheelEncoder;
+
```

Comment:
We'll need to add a `SparkPIDController` for both motors:

```suggestion
    private SparkPIDController _upperPIDController;
    private SparkPIDController _lowerPIDController;
```